lua/stand.lua

```diff
@@ -44,7 +45,37 @@ function M.setup(options)
 
 		timer:stop()
 		timer:again()
-	end, {})
+	end, { "Stand, dismissing any notifications and restarting the timer." })
+
+	vim.api.nvim_create_user_command("StandEvery", function(opts)
+		options.minute_interval = tonumber(opts.args) * 60000
+		local remaining = timer:get_due_in()
+		timer:stop()
+		timer:start(remaining, options.interval, vim.schedule_wrap(on_timer)) -- Start timer with new time
```

Comment:
Haven't tested, but I believe this should be `options.minute_interval`?

```suggestion
		timer:start(remaining, options.minute_interval, vim.schedule_wrap(on_timer)) -- Start timer with new time
```

Reply:
You're right! Somehow I missed this.